debezium-core/src/main/java/io/debezium/relational/history/DatabaseHistory.java

```diff
@@ -146,7 +158,17 @@
      */
     void initializeStorage();
 
-    boolean storeOnlyMonitoredTables();
+    boolean storeOnlyCapturedTables();
 
     boolean skipUnparseableDdlStatements();
+
+    static int validateMonitoredTables(Configuration config, Field field, Field.ValidationOutput problems) {
+        String msg = config.getFallbackStringPropertyWithWarning(STORE_ONLY_CAPTURED_TABLES_DDL, STORE_ONLY_MONITORED_TABLES_DDL);
```

Comment:
Won't this raise the warning too if only STORE_ONLY_CAPTURED_TABLES_DDL has been given? I think you'd want to only query for STORE_ONLY_MONITORED_TABLES_DDL, without any fallback?

Reply:
The warning will be only emitted when both/only `STORE_ONLY_MONITORED_TABLES_DDL` is provided as this validation check is added to the deprecated field only.